### DISCUSSION TITLE
Failure when loading animation and track ds desert hill, low range cell phone

### DISCUSSION THREAD

**Discussion by Skybloomerskullfire** [Medard22/Dolphin-MMJR2-VBI#877] [Bug Report]
![Screenshot_20241226-004419](https://github.com/user-attachments/assets/909c9f3e-43fb-4dc9-92e9-456b1691fe44)

**Comment by Medard22** [comment]
Same as your other post. Use English and don't duplicate bug reports. Thanks.
